funcions/11.test.js

```diff
@@ -0,0 +1,12 @@
+const { say } = require('./11')
+
+describe('say', () => {
+	const val1 = 'Hello, '
+	const val2 = 'it’s me'
+	it(`retrun 'Hello, it's me' when called with provided values`, () => {
```

Comment:
Looks like a typo. Consider installing spell-checker plugin for VSCode ;)

Reply:
Spell-checker is definitely the thing I needed :D